hms-app/src/pages/Archive.tsx

```diff
@@ -3,22 +3,26 @@ import HackathonDetails from '../components/card-details/HackathonDetails'
 import HackathonSelectDropdown from '../components/HackathonSelectDropdown'
 import { Title } from '@mantine/core'
 
-export default function Archive() {
-  const [selectedHackweek, setSelectedHackweek] = useState('')
+enum Enum {
```

Comment:
this enum should be imported from the HackathonSelectDropdown and not recreated here